testsuite/features/secondary/srv_power_management.feature

```diff
@@ -101,6 +101,3 @@ Feature: IPMI Power management
 
   Scenario: Cleanup: remove remaining systems from SSM after power management tests
     When I click on the clear SSM button
-
-  Scenario: Check for errors in Cobbler monitoring
-    Then the local logs for Cobbler should not contain errors
```

Comment:
Why we remove this check?

Reply:
Probably an error of mine, sorry. I'll resurrect those lines.

Reply:
Done